socialite/src/components/Recommend/Recommend.js

```diff
@@ -0,0 +1,65 @@
+import React, { useContext } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+import { AuthContext } from '../../context/auth'
+
+import "../Dashboard/Dashboard.css"
```

Comment:
No need to import Dashboard.css, it gets automatically imported, since it is already called in a sibling component.